packages/ckeditor5-table/docs/features/table.md

```diff
@@ -338,6 +354,9 @@ ClassicEditor
 	.then( ... )
 	.catch( ... );
 ```
+<info-box>
+	Check the {@link framework/guides/creating-simple-plugin plugin development} guide if you need more information about the technical side of this solution.
```

Comment:
```suggestion
	Check the {@link framework/guides/creating-simple-plugin plugin development guide} if you need more information about the technical side of this solution.
```

Reply:
We used to leave the and guide outside the link?